continuous_integration/environment-3.10-jdk11-dev.yaml

```diff
@@ -0,0 +1,43 @@
+name: dask-sql
+channels:
+- conda-forge
+- defaults
+dependencies:
+- adagio>=0.2.3
+- antlr4-python3-runtime>=4.9.2
+- black=19.10b0
+- ciso8601>=2.2.0
+- dask-ml>=2022.1.22
+- dask>=2021.11.1
+- fastapi>=0.61.1
+- fs>=2.4.11
+- intake>=0.6.0
+- isort=5.7.0
+- jpype1>=1.0.2
+- lightgbm>=3.2.1
+- maven>=3.6.0
+- mlflow>=1.19.0
+- mock>=4.0.3
+- nest-asyncio>=1.4.3
+- numpy<1.22  # numba only supports up to numpy 1.21
```

Comment:
We shouldn't have to make this pinning - the `defaults`-hosted Numba package that our Conda environments pick up doesn't have the expected max version constraint on numpy, which makes it possible to install a conflicting 1.22+ version.

Opened up https://github.com/numba/numba/issues/7794 describing this issue.

Reply:
Resolved this by removing the `defaults` channel